### PR TITLE
Favicon generator: replace realfavicon with favicons package

### PR DESCRIPTION
**Pull request by isstuev**
## Description and Related Issue(s)

resolves https://github.com/blockscout/frontend/issues/2287

### Proposed Changes
*[Specify the changes or additions made in this pull request. Please mention if any changes were made to the ENV variables]*

### Breaking or Incompatible Changes
*[Describe any breaking or incompatible changes introduced by this pull request. Specify how users might need to modify their code or configurations to accommodate these changes.]*

### Additional Information
*[Include any additional information, context, or screenshots that may be helpful for reviewers.]*

## Checklist for PR author
- [ ] I have tested these changes locally.
- [ ] I added tests to cover any new functionality, following this [guide](./CONTRIBUTING.md#writing--running-tests)
- [ ] Whenever I fix a bug, I include a regression test to ensure that the bug does not reappear silently.
- [ ] If I have added, changed, renamed, or removed an environment variable
    - I updated the list of environment variables in the [documentation](ENVS.md) 
    - I made the necessary changes to the validator script according to the [guide](./CONTRIBUTING.md#adding-new-env-variable)
    - I added "ENVs" label to this pull request

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -102,6 +112,7 @@ COPY --from=builder /app/public ./public
 COPY --from=builder /app/package.json ./package.json
 COPY --from=builder /app/deploy/tools/envs-validator/index.js ./envs-validator.js
 COPY --from=builder /app/deploy/tools/feature-reporter/index.js ./feature-reporter.js
+# COPY /app/deploy/tools/favicon-generator/index.js ./favicon-generator.js
```
Comment (tom2drum) [inline Dockerfile:115]: Shall we remove it?

File: public/assets/favicon/apple-touch-icon-180x180.png
Comment (tom2drum) [inline public/assets/favicon/apple-touch-icon-180x180.png:1]: Is it ok that the background is not transparent anymore?

File: tools/scripts/favicon-generator.dev.sh [hunk 2]
```diff
@@ -10,7 +10,7 @@ fi
 dotenv \
   -v MASTER_URL=$master_url \
   -e $secrets_file \
-  -- bash -c 'cd ./deploy/tools/favicon-generator && ./script.sh'
+  -- bash -c 'cd ./deploy/tools/favicon-generator && node "$(dirname "$0")/index.js"'
```
Comment (tom2drum) [inline tools/scripts/favicon-generator.dev.sh:13]: ⬆️ I think we don't need to load secret ENVs anymore.

File: deploy/tools/favicon-generator/index.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      for (const file of result.files) {
+        if (file.name !== 'manifest.webmanifest') {
+          await fs.writeFile(path.join(outputDir, file.name), file.contents);
+        }
+      }
+
+      console.log('Favicons generated successfully!');
+    } catch (faviconError) {
+      console.warn('Error generating favicons:', faviconError);
+      console.warn('Skipping favicon generation. Please check your sharp installation.');
```
Comment (tom2drum) [inline deploy/tools/favicon-generator/index.js:63]: Is it a relevant log?

File: public/assets/favicon/apple-touch-icon-180x180.png
Comment (isstuev) [reply public/assets/favicon/apple-touch-icon-180x180.png:1]: can change back to transparent

**Review by tom2drum — APPROVED** [review]

**Pull request merged**